src/screens/Account/TokenContractAddress.js

```diff
@@ -25,9 +26,13 @@ const forceInset = { bottom: "always" };
 
 const TokenContractAddress = ({ account, onClose, url, token }: Props) => {
   const { colors } = useTheme();
+
+  const currencyId = account ? account.token.name : token ? token.name : "";
+
   const viewInExplorer = useCallback(() => {
     if (url) {
       Linking.openURL(url);
+      track("ViewInExplorer", { currencyId });
```

Comment:
Same issue here theres already an event on button click :shrug: `GoToViewContractInExplorer` same as the other one just add currencyId in eventProperties